e2e/scripts/utils.js

```diff
@@ -26,8 +26,76 @@ const diffArrays = (expectedArr, actualArr) => {
   return [...expectedArr].filter((x) => !actualSet.has(x));
 };
 
+const getCreditCardExpiry = (yearsFromNow = 5) => {
+  const padMonth = "00";
+  return `${(padMonth + (new Date().getMonth() + 1)).slice(-padMonth.length)}/${
+    (new Date().getFullYear() % 100) + parseInt(yearsFromNow)
+  }`;
+};
+
+/**
+ * Generates a random string of given length containing uppercase letters, lowercase letters and numbers.
+ * @param {number} length Length of generated string required.
+ * @returns Randomly generated alphanumeric string.
+ */
+var generateRandomString = function (length) {
```

Comment:
Is there a specific reason you're using `var`? If not, let's use `const` and `let` instead throughout this file